gobblin-runtime/src/main/java/gobblin/runtime/instance/DefaultGobblinInstanceDriverImpl.java

```diff
@@ -137,7 +137,8 @@ public JobSpecRunnable(JobSpec jobSpec) {
 
     @Override
     public void run() {
```

Comment:
This is actually a method of JobSpecRunnable which gets passed to the JobExecutionScheduler.